app/src/main/java/org/simple/clinic/phone/MaskedPhoneCallerImplementation.kt

```diff
@@ -21,7 +20,7 @@ class MaskedPhoneCallerImplementation @Inject constructor(
     return if (config.maskingEnabled) {
       val number = "07930684724"
 
-      val stopCharacter = Uri.encode("#")
+      val stopCharacter = "%23" // encoding for '#'
```

Comment:
Can we revert to using `Uri.encode` instead? There are many types of encoding, and using `Uri.encode` is a lot more explicit about what kind of encoding we are doing.

If you did this because we can't use the `Uri` class in the test, then we should create an interface, say `PhoneNumberEncoder` and then replace it in the test with something that can be used in the tests.

Reply:
Turns out we do not need encoding anymore. Plain "#" is what's required.